rocketpy/sensors/accelerometer.py

```diff
@@ -241,64 +250,26 @@ def measure(self, t, u, u_dot, relative_position, gravity, *args):
-        self._save_data((t, *A))
+        self._save_data((time, *A))
 
-    def export_measured_data(self, filename, format="csv"):
-        """
-        Export the measured values to a file
+    def export_measured_data(self, filename, file_format="csv"):
+        """Export the measured values to a file
 
         Parameters
         ----------
         filename : str
             Name of the file to export the values to
-        format : str
+        file_format : str
             Format of the file to export the values to. Options are "csv" and
             "json". Default is "csv".
 
         Returns
         -------
         None
         """
-        if format.lower() not in ["json", "csv"]:
-            raise ValueError("Invalid format")
-        if format.lower() == "csv":
-            # if sensor has been added multiple times to the simulated rocket
-            if isinstance(self.measured_data[0], list):
-                print("Data saved to", end=" ")
-                for i, data in enumerate(self.measured_data):
-                    with open(filename + f"_{i+1}", "w") as f:
-                        f.write("t,ax,ay,az\n")
-                        for t, ax, ay, az in data:
-                            f.write(f"{t},{ax},{ay},{az}\n")
-                    print(filename + f"_{i+1},", end=" ")
-            else:
-                with open(filename, "w") as f:
-                    f.write("t,ax,ay,az\n")
-                    for t, ax, ay, az in self.measured_data:
-                        f.write(f"{t},{ax},{ay},{az}\n")
-                print(f"Data saved to {filename}")
-            return
-        if format.lower() == "json":
-            if isinstance(self.measured_data[0], list):
-                print("Data saved to", end=" ")
-                for i, data in enumerate(self.measured_data):
-                    dict = {"t": [], "ax": [], "ay": [], "az": []}
-                    for t, ax, ay, az in data:
-                        dict["t"].append(t)
-                        dict["ax"].append(ax)
-                        dict["ay"].append(ay)
-                        dict["az"].append(az)
-                    with open(filename + f"_{i+1}", "w") as f:
-                        json.dump(dict, f)
-                    print(filename + f"_{i+1},", end=" ")
-            else:
-                dict = {"t": [], "ax": [], "ay": [], "az": []}
-                for t, ax, ay, az in self.measured_data:
-                    dict["t"].append(t)
-                    dict["ax"].append(ax)
-                    dict["ay"].append(ay)
-                    dict["az"].append(az)
-                with open(filename, "w") as f:
-                    json.dump(dict, f)
-                print(f"Data saved to {filename}")
-            return
+        export_sensors_measured_data(
+            sensor=self,
+            filename=filename,
+            file_format=file_format,
+            data_labels=("t", "ax", "ay", "az"),
+        )
```

Comment:
I don't know about this. The file_format I guess refers more to the way data is saved in the file. Someone could want a `.txt` file with the data written in the csv format or in the json format

For me, it seems simpler just to have this choice as an argument, but I understand it can be redundant information in certain cases

Reply:
> Someone could want a .txt file with the data written in the csv format or in the json format

This is not reasonable to me. I don't see why someone would try to do this.

I think it's more annoying having to write "filename.txt, file_format='xt'", I don't see why this is simpler to you.